prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/FHIRBundleHelpersTests.kt

```diff
@@ -142,9 +142,32 @@ class FHIRBundleHelpersTests {
         // assert
         val provenance = bundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
         val outs = provenance.target
-        val receiversOut = outs.map { (it.resource as Endpoint).identifier[0].value }
-        assert(receiversOut.isNotEmpty())
-        assert(receiversOut[0] == "co-phd.full-elr-hl7")
+        val receiversOut = outs.map { it.resource }
+            .filterIsInstance<Endpoint>().map { it.identifier[0].value }
+        assertThat(receiversOut.isNotEmpty())
+        assertThat(receiversOut[0] == "co-phd.full-elr-hl7")
+    }
```

Comment:
Thanks for updating these, just move the checks outside the assertThat()

Reply:
Assertions updated.